### PR TITLE
Install.rst: Added note for Ubuntu users

### PR DESCRIPTION
**Pull request by vaishnavm217**
Added note for not using apt for installing coala

Closes https://github.com/coala/documentation/issues/392

### CODE REVIEW

**Comment by incorrectusername** [comment]
Always Use imperative present tense in commit head. `Added` to `Add`

**Comment by incorrectusername** [comment]
Your commit doesn't match coala standards.http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#how-to-write-good-commit-messages.

**Comment by vaishnavm217** [comment]
omg i thought --amend will open my previous unsuccessful commit instead it got renamed that
i just need to do git rebase -i then and write my commit again?

**Comment by vaishnavm217** [comment]
oh

**Comment by sils** [comment]
Hey still not correct, don't give up please this isn't easy. Read our docs, use git tutorials, ask people on gitter (only after you tried on your own, you need to understand this anyway!) - we're here to help you.

**Comment by sils** [comment]
coala.io/commit not followed, marking wip

File: Users/Install.rst [hunk 1]
```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use **apt** command for installing coala
+    It's a different software
```
Comment (Adrianzatreanu) [inline Users/Install.rst]: i bump this comment. also, it should be "don't use ``apt-get install coala`` for installing coala as that's a different software." (note the trailing period)

File: Users/Install.rst [hunk 2]
```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use ``apt-get install coala`` for installing coala,
```
Comment (vaishnavm217) [inline Users/Install.rst]: FML just 1 character!! :cry:

Comment (Nosferatul) [inline Users/Install.rst]: yea, just change that 😄

Comment (Nosferatul) [inline Users/Install.rst]: also, rebase

File: Users/Install.rst [hunk 3]
```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala, that's a different software.
```
Comment (Adrianzatreanu) [inline Users/Install.rst]: alright so a few more things and this is done:
1. i think "as that's" sounds a bit better :)
2. commit body: "Added" -> "Adds"
"for not to using" -> "for not using"
end with period.

After this, it should be done :)

**Comment by Adrianzatreanu** [comment]
also, a rebase would be needed :)

**Comment by Udayan12167** [comment]
ack fe8cca8

**Comment by Udayan12167** [comment]
@rultor merge

**Comment by vaishnavm217** [comment]
Now it might work 🙏  ;_;

**Comment by Udayan12167** [comment]
Umm you have a merge commit. That is undesireable. Keep at it. You need to use rebase. It's just a couple of git commands.

File: Users/Install.rst [hunk 4]
```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala as that's a different software.
```
Comment (jayvdb) [inline Users/Install.rst]: "as that's a different software." is bad English

use

"as that is different software."

**Comment by sims1253** [comment]
can you squash your commits using  `git rebase -i HEAD~2`?
See http://coala.io/rebase for more info

**Comment by Adrianzatreanu** [comment]
ok this rebase has gone completely nuts, please fix it :D

**Pull request closed without merging**